nautobot/dcim/choices.py

```diff
@@ -821,6 +821,17 @@ class InterfaceTypeChoices(ChoiceSet):
     TYPE_E1 = "e1"
     TYPE_T3 = "t3"
     TYPE_E3 = "e3"
+    TYPE_EIA232= "eia232"
+    TYPE_EIA422 = "eia422"
+    TYPE_EIA423 = "eia423"
+    TYPE_EIA485 = "eia485"
+    TYPE_EIA485_modbus = "modbus"
+    TYPE_EIA485_profibus = "profibus"
+    TYPE_fiber_profibus = "profibus-fiber"
+    TYPE_HART_RS232 = "hart-rs232"
+    TYPE_HART_RS422 = "hart-rs422"
+    TYPE_HART_RS485 = "hart-rs485"
+    TYPE_HART_bell202 = "hart-bell202"
```

Comment:
Our general convention would be for each of the constants to be all-caps, rather than mixed case, i.e. `TYPE_EIA485_MODBUS`, etc.